tsconfig.json

```diff
@@ -1,20 +1,27 @@
 {
+  "compileOnSave": false,
+  "include": ["src/**/*"],
+  "exclude": ["src/**/*.test.ts", "test/*"],
   "compilerOptions": {
+    "baseUrl": "./",
+    "module": "commonjs",
```

Comment:
How does this change impact consumers?

Does it work with when importing it from NodeJS, Webpack 4/5, etc?

Only asking because we had issues in the past importing in different environments

Reply:
The result looks like this:
https://github.com/cowprotocol/cow-sdk/blob/refactoring/dogfooding-dist/dist/order-book/api.js

Should work fine in browser env and in nodejs.

Reply:
Cool, but have you actually tested it?
Again, just asking because it happened in the past.

For example https://github.com/cowprotocol/cow-sdk/issues/71
Ok, that's issue is with app-data 😅 but there were more issues like these.

We said that we would add a test to verify it can be built and added to nodejs projects but we never got to implement it

Reply:
Tested with:
Explorer dogfooding: https://github.com/cowprotocol/explorer/pull/352
CowSwap dogfooding: https://github.com/cowprotocol/cowswap/pull/2071

Also I removed `app-data` dependency :)